Open.h

```diff
@@ -3,16 +3,206 @@
 #include "HashTable.h"
 #include "Product.h"
 #include <iostream>
-#include <map>
+#include <vector>
+#include <cmath>
+#include <string>
 
 template<typename Key, typename Value>
 class OpenMap: public HashTable<Key, Value> {
-    public:
-        OpenMap() = default;
+private:
+    double MAX_LOAD_FACTOR = 0.7;
```

Comment:
I like to open the class with `public `declarations.  Leave private for the bottom.